File: keychain-sample-chat-dot-net/KeychainChat/MVVM/Model/MessageType.cs
using KeychainChat.Common;

namespace KeychainChat.MVVM.Model
{
    public static class MessageType
    {
        public const string PairRequest = Constants.PAIR_REQUEST;
        public const string PairResponse = Constants.PAIR_RESPONSE;
        public const string PairAck = Constants.PAIR_ACK;
        public const string ChatMessage = Constants.CHAT_MESSAGE;
    }
}

File: keychain-sample-chat-iphone/KeychainChat/Models/MessageType.swift
//
//  PairMessageType.swift
//  consumer-app-iphone
//
//  Created by Robert Ellis on 2021/12/10.
//

import Foundation

enum MessageType: String, Codable {
    case pairRequest = "pairRequest"
    case pairResponse = "pairResponse"
    case pairAck = "pairAck"
    case chatMessage = "chatMessage"
}
